src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -67,23 +86,89 @@ const getInstallation = async (subscription: Subscription, log: Logger, gitHubAp
 	}
 };
 
-interface FailedConnection {
-	id: number;
-	deleted: boolean;
-	orgName?: string;
-}
+const getConnectionsAndInstallations = async (subscriptions: Subscription[], req: Request, id: number): Promise<ConnectionsAndInstallations> => {
+	const installations = await getInstallations(subscriptions, req.log, id);
+
+	const failedConnections: FailedConnection[] = await Promise.all(
+		installations.rejected.map(async (installation) => {
+			const sub = subscriptions.find((sub: Subscription) => installation.id === sub.gitHubInstallationId);
+			const repo = sub && await RepoSyncState.findOneFromSubscription(sub);
+			return {
+				id: installation.id,
+				deleted: installation.deleted,
+				orgName: repo?.repoOwner
+			};
+		}));
+
+	const successfulConnections: SuccessfulConnection[] = installations.fulfilled
+		.map((installation) => ({
+			...installation,
+			isGlobalInstall: installation.repository_selection === "all"
+		}));
+
+	return { installations, successfulConnections, failedConnections };
+};
 
-interface SuccessfulConnection extends AppInstallation {
-	isGlobalInstall: boolean;
-}
+const JiraCloudConfiguration = async (res: Response, req: Request): Promise<object> => {
+	const { jiraHost, gitHubAppId, nonce } = res.locals;
+	const subscriptions = await Subscription.getAllForHost(jiraHost);
+	const { installations, successfulConnections, failedConnections } = await getConnectionsAndInstallations(subscriptions, req, gitHubAppId);
+
+	return {
+		host: jiraHost,
+		successfulConnections,
+		failedConnections,
+		hasConnections: !!installations.total,
+		APP_URL: process.env.APP_URL,
+		csrfToken: req.csrfToken(),
+		nonce,
+		handleNavigationClassName: "add-organization-link",
+		isGitHubCloudApp: await isGitHubCloudApp(gitHubAppId)
```

Comment:
Shouldn't this be true by default for this function?

Reply:
This has been removed